form-validator/jquery.form-validator.js

```diff
@@ -265,7 +265,8 @@
         inputIsOptional = $elem.valAttr('optional') === 'true',
         skipBecauseDependingInputIsEmpty = false,
         skipBecauseItsEmpty = !value && inputIsOptional,
-        skipBecauseInputIsHidden = $elem.attr('disabled') || (!$elem.is(':visible') && !conf.validateHiddenInputs),
+        validationRules = $elem.attr(conf.validationRuleAttribute),
+        skipBecauseInputIsHidden = $elem.attr('disabled') || (!$elem.is(':visible') && !conf.validateHiddenInputs && (typeof validationRules !== typeof undefined && validationRules.indexOf("recaptcha") < 0)),
```

Comment:
I'm afraid this won't go, for two reasons.

1. This is the distribution file generated by grunt. The source files are located in /src/.
2. The core of this plugin should have as little knowledge of the validators as possible.

I will soon push a solution for this edge case. The modules will be able to bind to the event `beforeValidation` and tell the validator if the input should be skipped. The core library will do the same thing when determining if the input should be skipped because it's empty. So then we could in this particular case do something like:

```js
$('[data-validation~="recaptcha"]').on('beforeValidation', function(evt) {
  evt.stopImmediatePropagation(); // prevent other handler from setting this input as skipped because it's hidden
});
```

So please revert this file and I'll take care of it once I have pushed my solution to the master branch.